content/blog/2019-09-11-migrate-from-helm-v2-to-helm-v3.md

```diff
@@ -38,25 +38,26 @@ Let's install it:
 
 ```
 $ helm3 plugin install https://github.com/helm/helm-2to3
-Downloading and installing helm-2to3 v0.1.0 ...
-https://github.com/helm/helm-2to3/releases/download/v0.1.0/helm-2to3_0.1.0_darwin_amd64.tar.gz
+Downloading and installing helm-2to3 v0.1.2 ...
+https://github.com/helm/helm-2to3/releases/download/v0.1.0/helm-2to3_0.1.2_darwin_amd64.tar.gz
 Installed plugin: 2to3
 ```
 
 ```
 $ helm3 plugin list
 NAME	VERSION	DESCRIPTION
-2to3	0.1.0  	migrate Helm v2 configuration and releases in-place to Helm v3
+2to3	0.1.2  	migrate Helm v2 configuration and releases in-place to Helm v3
```

Comment:
migrate Helm v2 configuration and releases in-place to Helm v3 --> migrate and clean up Helm v2 configuration and releases in-place to Helm v3

We can update the plugin description in the code base subsequently

Reply:
done